src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -227,6 +231,48 @@ def list_volumes(mgr):
         result.append({'name': f['mdsmap']['fs_name']})
     return result
 
+def info(mgr, path, fs_handle, volname):
+    """
+    Get all metadata of a volume.
```

Comment:
Can use word `information` instead of `metadata`.

Reply:
Done